docs/samples/config/mscolab/mscolab_settings.py.sample

```diff
@@ -27,7 +27,11 @@
 import os
 import logging
 
+
 class mscolab_settings:
+    # In the unit months when Operations get archived because not used
```

Comment:
good to add it here too. i just looked how it is configured in the https://github.com/Open-MSS/MSS/blob/develop/mslib/mscolab/conf.py#L44

I think we should keep it in days. I did not described it correctly in the issue.

Please change it to days so that it matches with the other occurencies where it is already used.